### ISSUE TITLE
Dynamic import regression in the read-eval-print-loop

### ISSUE THREAD

**Issue by caspervonb** [denoland/deno#7786]
Introduced in #7763

```
> import("./")
Promise { <pending> }
> _
Promise {
  <rejected> TypeError: invalid base URL for relative import: relative URL without a base

}
```

**Comment by caspervonb** [comment]
Would appear that the context has no origin

```
{"method":"Runtime.executionContextCreated","params":{"context":{"id":1,"origin":"","name":"global context"}}}
```

**Comment by bartlomieju** [comment]
Fixed in #7794
